Octo-Bot/packet-generator-bot/bot/ReplayGenerator.py

```diff
@@ -0,0 +1,66 @@
+from scapy.all import *
+
+
+from PacketGenerator import *
+from helper.ReplayHelper import *
+from utils.TimeUtils import *
+
```

Comment:
Can you a little documentation for this function?
Including expected inputs/arguments.

Reply:
Added!